### ISSUE TITLE
ImportError: No module named flask

### ISSUE THREAD

**Issue by Bystroushaak** [FedericoCeratto/bottle-cork#79]
Hello.

Today, I installed the ``cork`` using ``pip install bottle-cork`` command:

```
bottle-cork               - Authentication/Authorization library for Bottle
  INSTALLED: 0.11 (latest)
```

When I try to run it, I get this error:

```python
>>> import cork
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/cork/__init__.py", line 7, in <module>
    from cork import Cork, JsonBackend, AAAException, AuthException, Mailer, FlaskCork, Redirect
  File "/usr/local/lib/python2.7/dist-packages/cork/cork.py", line 30, in <module>
    import flask
ImportError: No module named flask
```

**Comment by cperezp** [comment]
Same here. This happens to me on OSX (Mavericks), using Python, pip, Bottle, and Cork, all installed via Homebrew.

At home I have Yosemite installed, and did not experience the same issue.

**Comment by FedericoCeratto** [comment]
I'll fix it shortly!

**Comment by FedericoCeratto** [comment]
Closed in version 0.11.1
